### PR TITLE
Enhancement/8661 pax success banner

### PR DESCRIPTION
**Pull request by zutigrm**
## Summary

Addresses issue:

- #8661

## Relevant technical choices

* Note, the current VRT is the existing Ads setup success one, so it is expected `show me` button is not present. The PAX success notification only has stories, but it is not added to the VRT, as that part of the IB test section is crossed.

## PR Author Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 5.2 and PHP 7.4.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: assets/sass/components/dashboard/_googlesitekit-subtle-notification.scss [hunk 1]
```diff
@@ -61,6 +62,7 @@
 
 	.mdc-button {
 		margin: 0 auto;
+		min-height: 32px; // Default min-height across the plugin is 40px, but in this instance it shouldn't be that big.
```
Comment (tofumatt) [inline assets/sass/components/dashboard/_googlesitekit-subtle-notification.scss:65]: This comment doesn't explain _why_ it should be smaller, just that it should be, but that's already evident by the style declaration 😄 

The comment can be removed unless you need/want to explain the reasoning, if it's complicated 🙂 

```suggestion
		min-height: 32px;
```

**Review by tofumatt — CHANGES_REQUESTED** [review]
Largely looks good, just a note about dismissing the notification for both CTAs really 🙂

**Comment by zutigrm** [comment]
Thanks @tofumatt PR updated

**Review by tofumatt — APPROVED** [review]

**Review by tofumatt — COMMENTED** [review]
The PR here looks good but there are a few merge conflicts that need resolving, then we can merge 🙂

**Comment by zutigrm** [comment]
@tofumatt Conflicts fixed. E2E nightly seems to be failing for some unrelated test regarding admin-bar

**Review by tofumatt — APPROVED** [review]

**Pull request merged**